client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
```

Comment:
prefix should be passed as a constructor parameter. it can default to the empty string.